privaterelay/templates/profile.html

```diff
@@ -54,11 +54,11 @@
   {% endif %}
   <div class="main-list flx flx-col jst-cntr">
   {% if messages %}
-    <div class="messages profile-messages hidden">
+    <div class="messages hidden">
```

Comment:
Could you revise the `.hidden` class to `.is-hidden` while you're making these changes?